docs/concepts/vega-chain/transactions.md

```diff
@@ -0,0 +1,87 @@
+---
+sidebar_position: 4
+title: Transactions
+vega_network: TESTNET
+hide_title: false
+---
+
+import NetworkParameter from '@site/src/components/NetworkParameter';
+
+A transaction is an instruction (command) issued the network, bundled with a signature, the submitter’s public key, a small proof-of-work calculation, and the target block height. A transaction that doesn't pass [validation](#validation) (for example, an order without enough assets to cover it) will be rejected.
+
+:::tip Viewing transactions
+Use the [Vega block explorer](https://explorer.fairground.wtf) to see transactions and their status, plus lots more data about the network.
+:::
+
+When a transaction is submitted, it goes to the validators’ mempool - which is is an organised queue where the transactions are stored, sorted, and verified before being added to a newly started block. A validated transaction enters a block based on the priority assigned to the transaction type, and the order in which it arrived.
+
+## Commands
+A command is the specific instruction issued to the network within a transaction.
+
+The commands that non-validator users can be submitted to the network include submitting, amending, and cancelling orders, liquidity provisions and transfers, associating tokens and nominating validators, governance proposals and voting on proposals, and depositing and withdrawing assets.
+
+Validators can submit signatures, propose protocol upgrades, rotate their keys and more.
+
+:::tip Query for data
+See the gRPC docs for full lists of [user commands](../../api/grpc/vega/commands/v1/commands.proto) and [validator commands](../../api/grpc/vega/commands/v1/validator_commands.proto) supported by the protocol.
+:::
+
+
+## Validation
+Transactions are validated before they are added to the mempool, and then again once they’re in the mempool but before they are executed. 
+
+A transaction may be rejected throughout the validation process for several reasons, including because it doesn’t pass the spam check, it is malformed or missing information, or conditions have made it irrelevant (such as not enough of an asset to fulfil an order). 
```

Comment:
```suggestion
A transaction may be rejected throughout the validation process for several reasons, including because it doesn’t pass the spam check, it is malformed or missing information, or conditions have made it irrelevant (for example if a user's balance changes and they no longer have the balance to fulfil an order). 
```

**nitpick (non-blocking)**:  found this parenthetical a little clunky, so I proposed an alternative.